### PR TITLE
Release Emilia and Emilia-Pipe

### PR DESCRIPTION
**Pull request by yuantuo666**
## ✨ Description
We release the **Emilia**, an extensive, multilingual, and diverse dataset, and **Emilia-Pipe**, the first open-source preprocessing pipeline designed to transform in-the-wild speech data into high-quality training data with annotations for speech generation.

Major contribution for this PR: @HarryHe11 @shangqwe123 @yuantuo666 @lixuyuan102

## 🚧 Related Issues

None

## 👨‍💻 Changes Proposed

- [x] Update `README.md` News
- [x] Add a `README.md` in `preprocessors/Emilia` directory to introduce Emilia-Pipe
- [x] Integrate processing pipeline Emilia-Pipe under `preprocessors/Emilia` directory

## 🧑‍🤝‍🧑 Who Can Review?

@HarryHe11 @jiaqili3 @RMSnow @HeCheng0625 

## 🛠 TODO

None

## ✅ Checklist

- [ ]  Code has been reviewed
- [x]  Code complies with the project's code standards and best practices
- [x]  Code has passed all tests
- [x]  Code does not affect the normal use of existing features
- [x]  Code has been commented properly
- [x]  Documentation has been updated (if applicable)
- [x]  Demo/checkpoint has been attached (if applicable)

### CODE REVIEW

**Comment by HarryHe11** [comment]
Hi Chaoren @yuantuo666 , thank you so much for raising this PR and all of your efforts @yuantuo666  @shangqwe123 @lixuyuan102! 

I notice that we haven't provide the list for the source audios yet in this pr. Maybe we could complete this part before merging!

File: preprocessors/Emilia/README.md
Comment (HarryHe11) [inline preprocessors/Emilia/README.md:28]: @lixuyuan102 Hi xuyuan, could you please help add the links to all the used pre-trained models here under line 28?

File: preprocessors/Emilia/README.md
Comment (HarryHe11) [inline preprocessors/Emilia/README.md:93]: @shangqwe123 Dr. Shang, I think we'd better add a citation section to recommend the user of the repos cite Amphion and Emilia (after the arxiv release)

File: preprocessors/Emilia/models/dnsmos.py
Comment (HarryHe11) [inline preprocessors/Emilia/models/dnsmos.py:1]: @yuantuo666 Chaoren, I think we should double-check this copyright in this file. I think the dnsmos.py file is modified from other repos, we should give reference to this repos in the copyright statements

File: preprocessors/Emilia/models/separate_fast.py
Comment (HarryHe11) [inline preprocessors/Emilia/models/separate_fast.py:4]: @yuantuo666  same as my comments above.

File: preprocessors/Emilia/models/silero_vad.py
Comment (HarryHe11) [inline preprocessors/Emilia/models/silero_vad.py:2]: @yuantuo666  same as my comments above.

**Review by HarryHe11 — COMMENTED** [review]
@yuantuo666 @lixuyuan102 @shangqwe123

Thanks so much for Chaoren's efforts in preparing this final version of Emilia-Pipe, @yuantuo666!  I think most of the code is of very good quality, neat, and well-commented! 

Before merging this PR, I think we should provide the lists for the source audios and address my comments, as well as the important comments from Xueyao, Jiaqi, and Yuancheng. @RMSnow @jiaqili3 @HeCheng0625

File: README.md
Comment (RMSnow) [inline README.md:36]: 1. We should use more easy-to-understand words for news. People will be confused because they know little about "Emilia" -- What is this and what is it for? Why is it unique?
2. Use some badges like the following news to link some userful info.

File: README.md
Comment (RMSnow) [inline README.md:83]: 1. The URL `https://github.com/open-mmlab/Amphion/tree/main/preprocessors/Emilia` can not be accessed now. We should use a relative link for all our repo's material. Egs: `The supported dataset list can be seen [here](egs/datasets/README.md)`
2. Use itemsize to describe for Datasets section: 
```
1. Amphion unifies the data preprocess of the open-source datasets including [AudioCaps](https://audiocaps.github.io/), [LibriTTS](https://www.openslr.org/60/), [LJSpeech](https://keithito.com/LJ-Speech-Dataset/), [M4Singer](https://github.com/M4Singer/M4Singer), [Opencpop](https://wenet.org.cn/opencpop/), [OpenSinger](https://github.com/Multi-Singer/Multi-Singer.github.io), [SVCC](http://vc-challenge.org/), [VCTK](https://datashare.ed.ac.uk/handle/10283/3443), and more. The supported dataset list can be seen [here](egs/datasets/README.md) (updating).
2. Emilia's description: ...
```

File: preprocessors/Emilia/README.md
Comment (RMSnow) [inline preprocessors/Emilia/README.md:3]: There is a lack of high-level description about Emilia. For example, what is it source from? Why is it unique? How many hours is it? Some other metainfo (#speakers, #language, ...)? What can it be used for? ...

File: preprocessors/Emilia/config.json
Comment (RMSnow) [inline preprocessors/Emilia/config.json:19]: Do users need to modify the path? If so, add some comments. An example: [https://github.com/open-mmlab/Amphion/tree/main/egs/svc/MultipleContentsSVC#configuration](https://github.com/open-mmlab/Amphion/tree/main/egs/svc/MultipleContentsSVC#configuration)

File: preprocessors/Emilia/env.sh
Comment (RMSnow) [inline preprocessors/Emilia/env.sh:1]: Add the copyright information.

**Review by RMSnow — CHANGES_REQUESTED** [review]
Great works! The most important thing is to add some high-level descriptions for Emilia.

**Comment by HarryHe11** [comment]
@RMSnow Hi Xueyao, thank you so much for your helpful suggestions! I have addressed all of your comments accordingly!

File: preprocessors/Emilia/models/dnsmos.py
Comment (yuantuo666) [reply preprocessors/Emilia/models/dnsmos.py:1]: Sure, I will change to following statements:
```
# Source: https://github.com/microsoft/DNS-Challenge/tree/master/DNSMOS
#
# Copyright (c) 2022 Microsoft
# 
# This code is licensed under the Creative Commons Attribution 4.0 International (CC BY 4.0) license.
# The full license text is available at the root of the source repository.
#
# Note: This code has been modified to fit the context of this repository. 
#       This code is included in an MIT-licensed repository.
#       The repository's MIT license does not apply to this code. 
```

File: preprocessors/Emilia/models/separate_fast.py
Comment (yuantuo666) [reply preprocessors/Emilia/models/separate_fast.py:4]: https://github.com/seanghay/uvr-mdx-infer The source code did not claim the license. I temporaily added a MIT license here since the https://github.com/Anjok07/ultimatevocalremovergui repository claimed to be MIT-license.

Comment (yuantuo666) [reply preprocessors/Emilia/models/separate_fast.py:4]: We might need to requests a issue to original repository to ask about the license and relevent right about this.

File: preprocessors/Emilia/README.md
Comment (lixuyuan102) [inline preprocessors/Emilia/README.md:38]: Bgm Separator:[UVR-MDX-NET-Inst_HQ_3](https://github.com/TRvlvr/model_repo/releases/tag/all_public_uvr_models)
VAD:[Silero](https://github.com/snakers4/silero-vad)
SpeakerDiarization: [pyannote](https://github.com/pyannote/pyannote-audio)
ASR: [whisperx-medium](https://github.com/m-bain/whisperX)
AutoMOS:[DNSMOS P. 835](https://github.com/microsoft/DNS-Challenge)

File: preprocessors/Emilia/README.md
Comment (lixuyuan102) [reply preprocessors/Emilia/README.md:38]: @HarryHe11 Hi Haorui, the links are added in lines 30-43.

Comment (HarryHe11) [reply preprocessors/Emilia/README.md]: Thx!

File: preprocessors/Emilia/models/separate_fast.py
Comment (HarryHe11) [reply preprocessors/Emilia/models/separate_fast.py]: we can claim that ``this code is modified from xxxx.py`` like the following example: https://github.com/open-mmlab/Amphion/blob/5dfe9fd663a96f69854bb060fa52fe15946fe690/models/tts/valle/valle.py#L6

**Comment by HarryHe11** [comment]
@yuantuo666 @shangqwe123 @lixuyuan102 Chaoren, Zengqiang, Xuyuan, 

I think we'd better merge this pr after we got our arxiv link, and done all of the following tasks.

- [ ] Codes are up to date (all tasks are completed)
- [ ] Codes are tested. 
- [ ] Demopage is up to date (the abstract is consistent with our arxiv paper).
- [ ] Licenses and copyright declarations have been appropriately added in Github, huggingface.
- [ ] Huggingface releases the meta-information for the source audios.
- [ ] Amphion's main README relayed the news of Emilia's release and provides a link to the sub-repo.
- [ ] Emilia's README introduces the dataset adequately.
- [ ] Emilia's README provides appropriate URLs to the demopage. 2. Hugging Face Page. 3. Arxiv paper
- [ ] Emilia's README, Demopage, and huggingface display references to Emilia's arxiv work and amphion.

**Pull request merged**
